src/libcprover-cpp/api.cpp

```diff
@@ -168,6 +171,60 @@ void api_sessiont::verify_model() const
   verifier.report();
 }
 
+// TODO: This is a temporary function - it's basically `verify_model`, tweaked
```

Comment:
💡 I'd prefer documentation in the .h file because this is where the interface is defined.

Reply:
This is an actual TODO comment rather than documentation. It is explaining the need for clean-up rather than documenting what the function is supposed to be for. Therefore I would argue that this information belongs in the `.cpp` rather the header. There is a separate argument that there should be a doxygen block in the header which  deprecates this function, explains the difference from the `verify_model` function and that the `verify_model` function should be used in preference.